### DISCUSSION TITLE
Missing status icons after upgrade

### DISCUSSION THREAD

**Discussion by nickjupp** [teslamate-org/teslamate#3086] [Q&A] [answered]
Hi, just upgraded to 1.27.2 and now don't have the icons showing locked/plugged in etc on the home screen (although I do have a ❄️  for low temp!)
Anyone else having this... any ideas to rectify please?

thanks
Nick

**Comment by cwanja** [comment, marked as answer]
No issues here. On 1.27.2. Have you tried restarting the containers and or launching the Tesla app to ensure connectivity with TeslaMate?

**Reply by nickjupp** [reply]
Thanks, all good now after waking the car up!
